src/pagination/pagination.ts

```diff
@@ -179,7 +194,7 @@ export class NgAisPagination extends BaseWidget {
 
   public ngOnInit() {
     this.createWidget(connectPagination, {
-      maxPages: parseNumberInput(this.totalPages),
+      totalPages: parseNumberInput(this.totalPages),
```

Comment:
what did this change for? it looks correct, but not sure why the mapping would have been done initially?

Reply:
also why are other props like padding not passed?

Reply:
> also why are other props like padding not passed?
it looks like padding is managed inside [the widget layer](https://github.com/algolia/angular-instantsearch/blob/master/src/pagination/pagination.ts#L133-L164)
This is probably something we'll need to change, but I thought that could be done as non breaking change later

Reply:
> what did this change for? it looks correct, but not sure why the mapping would have been done initially?

This is very likely something a change I forgot when [upgrading from IS.js v2 to v3](
https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/js/#pagination-1).
I don't think it ever came up as an issue, up until using the actual types.

Reply:
no worries, this is good then!